main.py

```diff
@@ -38,8 +40,10 @@ def main():
     )
 
     defaults = Defaults(parse_mode=ParseMode.MARKDOWN)
-    updater = Updater(token=config["TELEGRAM_BOT_TOKEN"], use_context=True, defaults=defaults)
+    updater = Updater(token=config["TELEGRAM_BOT_TOKEN"],
+                      use_context=True, defaults=defaults)
```

Comment:
Either keep this a single line or format this properly
```suggestion
    updater = Updater(
            token=config["TELEGRAM_BOT_TOKEN"], use_context=True, defaults=defaults
    )
```